packages/core/src/main/util/encryptedStore.test.node.ts

```diff
@@ -39,7 +39,7 @@ describe('encryptedStore', () => {
 
   describe('Store and restore secret values with custom encryption', () => {
     it('Stores secret values', async () => {
-      const store = await createCustomEncryptedStore('test', {
+      const store = await createCustomEncryptedStore('test-custom', {
```

Comment:
unrelated fix that fixes a flaky test (both tests were accessing the same DB with different schema)